// react-browser/lib/components/Button/Button.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = exports.StyledButton = void 0;

var _react = _interopRequireDefault(require("react"));

var _propTypes = _interopRequireDefault(require("prop-types"));

var _styledComponents = _interopRequireDefault(require("styled-components"));

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

var StyledButton = _styledComponents.default.button.withConfig({
  displayName: "Button__StyledButton",
  componentId: "sc-1y61900-0"
})(["", ";"], function (props) {
  return props.theme.Button;
});

exports.StyledButton = StyledButton;

var Button = function Button(props) {
  return /*#__PURE__*/_react.default.createElement(StyledButton, props);
};

Button.propTypes = process.env.NODE_ENV !== "production" ? {
  /** Button content */
  children: _propTypes.default.node.isRequired,

  /** Button type */
  type: _propTypes.default.oneOf(['primary', 'secondary']),

  /** Enable outline buttons */
  outline: _propTypes.default.bool
} : {};
Button.defaultProps = {
  type: 'primary',
  outline: false
};
var _default = Button;
exports.default = _default;
