// react-browser/es/components/Button/Button.js
var _jsxFileName = "/Users/brians/git/react-browser/src/components/Button/Button.jsx",
    _this = this;

function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

import React from 'react';
import PropTypes from 'prop-types';
import styled from 'styled-components';
export var StyledButton = styled.button.withConfig({
  displayName: "Button__StyledButton",
  componentId: "sc-1y61900-0"
})(["", ";"], function (props) {
  return props.theme.Button;
});

var Button = function Button(props) {
  return /*#__PURE__*/React.createElement(StyledButton, _extends({}, props, {
    __self: _this,
    __source: {
      fileName: _jsxFileName,
      lineNumber: 8,
      columnNumber: 25
    }
  }));
};

Button.propTypes = process.env.NODE_ENV !== "production" ? {
  /** Button content */
  children: PropTypes.node.isRequired,

  /** Button type */
  type: PropTypes.oneOf(['primary', 'secondary']),

  /** Enable outline buttons */
  outline: PropTypes.bool
} : {};
Button.defaultProps = {
  type: 'primary',
  outline: false
};
export default Button;
